<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img/img.qrc">
    <normaloff>:/imga/lista.png</normaloff>:/imga/lista.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="locale">
    <locale language="Spanish" country="Argentina"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>8</number>
         </property>
         <property name="topMargin">
          <number>8</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>8</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,9,2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="crearCarpeta">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string notr="true">Crear Carpeta</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>img/nuevacarpeta.png</normaloff>img/nuevacarpeta.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>45</width>
               <height>50</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="autoRepeat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,9,2">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="abrirCarpeta">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Abrir Carpeta</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>img/abrircarpeta.png</normaloff>img/abrircarpeta.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>45</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,9,2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="guardar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Guardar Carpeta</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>img/guardarcarpeta.png</normaloff>img/guardarcarpeta.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>45</width>
               <height>50</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="title">
            <string>Datos Egresados</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,2,2,1,1">
            <item>
             <widget class="QLineEdit" name="nombre">
              <property name="inputMask">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Nombre del Egresado</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="padre">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Padre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="madre">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Madre</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="tarjetas">
              <property name="placeholderText">
               <string>Tarjetas Vendidas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="agregarBoton">
              <property name="text">
               <string>Agregar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editarBoton">
              <property name="text">
               <string>Editar</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="8,2">
           <item>
            <widget class="QLabel" name="nombreColegio">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Colegio</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="verInvitados">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Ver Invitados</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="tabla_egresados">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>124</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Egresado</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>1</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>3</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>4</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>5</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>6</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>7</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>8</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>9</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>10</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>11</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>12</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Padres/Tutores</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Tarjetas</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
